Widgets/RecentDonations.html

```diff
@@ -0,0 +1,35 @@
+<html>
+	<head>
+		<script type="text/javascript" src="../bower_components/jquery/dist/jquery.min.js"></script>
+		<script type="text/javascript" src="../bower_components/angular/angular.min.js"></script>
+		<script type="text/javascript" src="../js/Globals.js"></script>
+		<script type="text/javascript" src="../js/RecentDonations.js"></script>
+
+		<link href="http://fonts.googleapis.com/css?family=Cantarell:700" rel="stylesheet" type="text/css">
+
+		<link rel="stylesheet" type="text/css" href="widget.css">
+	</head>
+	<body>
+		<div id="widget" ng-app="RecentDonations" ng-controller="RecentDonationsControl">		
+
+			<h1>Recent Donations for {{ displayName }} </h1>
+
+			<div id="error" ng-show="error">{{ error }}</div>
+
+			<table ng-hide="error">
+		    <tr>
+				<th>Name</th>
+				<th>Amount</th>
+				<th>Message</th>
+				<th>Date</th>
+			</tr>
+			  <tr ng-repeat="x in contributors">
+			    <td>{{ x.name }}</td>
+			    <td>{{ x.amount }}</td>
+			    <td>{{ x.message }}</td>
```

Comment:
Not sure if I'd want to see the message here.

Similar widget/things on other streams are usually a horizontally scrolling text box with the name and the amount. If people leave long messages this can get very cluttered very quickly, and I'm not sure how interesting the exact time of the donation is (except for determining the most recent X donations).